Spanner/tests/Unit/Batch/BatchSnapshotTest.php

```diff
@@ -254,6 +278,14 @@ public function testExecutePartitionInvalidType()
         $dummy = new DummyPartition;
         $this->snapshot->executePartition($dummy);
     }
+
+    public function optionsProvider()
+    {
+        return [
+            [[]],
+            [['dataBoostEnabled' => true]]
```

Comment:
Not very useful, but can also add a `dataBoostEnabled : false` case.

Reply:
Yes I agree. Fixed.